src/Examples/Advanced/CustomProperties/Hydrocarbon_Processing_example.ipynb

```diff
@@ -413,54 +506,50 @@
-      "   4  0.0000000e+00 2.10e+01 6.92e+03  -1.0 1.62e+03    -  6.45e-01 9.90e-01h  1\n",
-      "   5  0.0000000e+00 2.17e-01 8.71e+03  -1.0 3.46e+02    -  9.90e-01 1.00e+00h  1\n",
-      "   6  0.0000000e+00 4.28e-03 4.45e+04  -1.0 4.81e+01    -  9.95e-01 1.00e+00h  1\n",
-      "   7  0.0000000e+00 1.17e-06 1.68e+01  -1.0 7.94e-01    -  1.00e+00 1.00e+00h  1\n",
+      "   0  0.0000000e+00 5.59e+04 1.00e+00  -1.0 0.00e+00    -  0.00e+00 0.00e+00   0\n",
+      "   1  0.0000000e+00 5.60e+02 6.70e-01  -1.0 1.50e+02    -  9.90e-01 9.90e-01h  1\n",
+      "   2  0.0000000e+00 5.56e+00 6.28e-01  -1.0 1.48e+02    -  9.90e-01 9.90e-01h  1\n",
+      "   3  0.0000000e+00 2.71e-04 9.84e+02  -1.0 1.46e+00    -  9.90e-01 1.00e+00h  1\n",
+      "   4  0.0000000e+00 9.31e-10 2.99e-03  -1.0 7.05e-05    -  1.00e+00 1.00e+00h  1\n",
       "\n",
-      "Number of Iterations....: 7\n",
+      "Number of Iterations....: 4\n",
       "\n",
       "                                   (scaled)                 (unscaled)\n",
       "Objective...............:   0.0000000000000000e+00    0.0000000000000000e+00\n",
       "Dual infeasibility......:   0.0000000000000000e+00    0.0000000000000000e+00\n",
-      "Constraint violation....:   4.0239024785785836e-09    1.1702068150043488e-06\n",
+      "Constraint violation....:   3.6379788070917130e-12    9.3132257461547852e-10\n",
       "Complementarity.........:   0.0000000000000000e+00    0.0000000000000000e+00\n",
-      "Overall NLP error.......:   4.0239024785785836e-09    1.1702068150043488e-06\n",
+      "Overall NLP error.......:   3.6379788070917130e-12    9.3132257461547852e-10\n",
       "\n",
       "\n",
-      "Number of objective function evaluations             = 8\n",
-      "Number of objective gradient evaluations             = 8\n",
-      "Number of equality constraint evaluations            = 8\n",
+      "Number of objective function evaluations             = 5\n",
+      "Number of objective gradient evaluations             = 5\n",
+      "Number of equality constraint evaluations            = 5\n",
       "Number of inequality constraint evaluations          = 0\n",
-      "Number of equality constraint Jacobian evaluations   = 8\n",
+      "Number of equality constraint Jacobian evaluations   = 5\n",
       "Number of inequality constraint Jacobian evaluations = 0\n",
-      "Number of Lagrangian Hessian evaluations             = 7\n",
-      "Total CPU secs in IPOPT (w/o function evaluations)   =      7.719\n",
-      "Total CPU secs in NLP function evaluations           =     43.892\n",
+      "Number of Lagrangian Hessian evaluations             = 4\n",
+      "Total CPU secs in IPOPT (w/o function evaluations)   =      7.190\n",
+      "Total CPU secs in NLP function evaluations           =     23.409\n",
       "\n",
-      "EXIT: Optimal Solution Found.\n",
-      "\b"
+      "EXIT: Optimal Solution Found.\n"
```

Comment:
This has nothing to do with the review of the changes, but I just thought it was interesting to look at as a relatively clean diff of the changes from the solver's output's point of view. Aside from this, I'm not sure if there's any useful information to be found by a more expert eye.

Reply:
The diff of the solve logs show that the newer version actually solves better than the previous one (which is what we would hope for with the addition of scaling and intermediate log variables).